src/app/components/DappBanner/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC } from 'react'
+import Card from '@mui/material/Card'
+import CardContent from '@mui/material/CardContent'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../styles/theme/colors'
+import Box from '@mui/material/Box'
+import { useTranslation } from 'react-i18next'
+import { SearchScope } from '../../../types/searchScope'
+import { getDappForAddress } from '../../config/dapps'
+import Button from '@mui/material/Button'
+
+export const DappBanner: FC<{ scope: SearchScope; address: string | undefined }> = ({ scope, address }) => {
+  const { t } = useTranslation()
+
+  if (!address) {
+    return null
+  }
+
+  const dApp = getDappForAddress(t, scope.network, scope.layer, address)
+
+  return (
+    !!dApp && (
+      <Card
+        sx={{
+          backgroundColor: COLORS.brandMedium,
+        }}
+      >
+        <CardContent>
+          <Box
+            sx={{
+              display: 'flex',
+              flexWrap: 'wrap',
+              justifyContent: 'center',
+              alignItems: 'center',
+              gap: 3,
+            }}
+          >
+            <Typography
+              variant="h3"
+              sx={{
+                color: COLORS.white,
+                fontFeatureSettings: `'clig' off, 'liga' off`,
+                fontSize: '24px',
+                fontStyle: 'normal',
+                fontWeight: 700,
+                lineHeight: '140%' /* 33.6px */,
+              }}
+            >
+              {dApp.description}
+            </Typography>
+            &nbsp;
+            <Button
+              href={dApp.url}
+              sx={{
+                backgroundColor: COLORS.white,
+                fontFeatureSettings: `'clig' off, 'liga' off`,
```

Comment:
whats this

Reply:
No idea. Copied it from Figma. I see that browsers _do_ parse it, but I don't see any difference in the rendered font image. I assume it's there in Figma with a reason. @donouwens should be able to deny / confirm.

Reply:
Never mind. Removed.